lib-api/src/main/java/org/ergoplatform/appkit/BlockchainDataSource.java

```diff
@@ -0,0 +1,66 @@
+package org.ergoplatform.appkit;
+
+import org.ergoplatform.restapi.client.BlockHeader;
+import org.ergoplatform.restapi.client.ErgoTransaction;
+import org.ergoplatform.restapi.client.ErgoTransactionOutput;
+import org.ergoplatform.restapi.client.NodeInfo;
+import org.ergoplatform.restapi.client.WalletBox;
```

Comment:
This is not good. 
Those classes and their structure is very specific to node implementation. 
The naming is a bit chaotic and not always consistent because API undergone many iterations over time.
It is often hard to decide what should be used and when.
The model classes are also bound to API methods and often duplicated.

Appkit interfaces on the other hand should be well thought.
It is ok if there is not one-to-one correspondence between Appkit data model and Node API+ExplorerAPI data models.
Moreover, Appkit should have single data model of all blockchain data available through either Node API or Explorer API or whatever API.

If something is missing in existing Appkit interfaces, then can be added either new method/property or new interface representing the corresponding conceptual entity.
So, basically, first conceptual model of Appkit should be extended on the level of interfaces, and only then those interfaces can be implemented in `impl` package in one way or another.